tools/BuildToolset.proj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)/BuildToolsetVersions.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <NonShipping>true</NonShipping>
+    <NoStdLib>true</NoStdLib>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="$(BuildTasksFeedToolVersion)" />
```

Comment:
Can you move this PackageReference into PublishToBlobFeed.proj, have this file import that project, and then use  BuildTools.proj for your entry point for everything (`dotnet restore`, `dotnet msbuild`) instead of using BuildToolset.proj for restore and PublishToBlobFeed for msbuild?

Reply:
PublishToBlobFeed.proj imports props and targets from the nuget package - therefore, the package needs to exist before the import takes place. Does a 'restore' suppress the importations?

Reply:
Wouldn't you be able to remove those imports and they'd be automatically imported via the packagereference?  I haven't experimented, but I thought that was part of what https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#including-msbuild-props-and-targets-in-a-package was alluding to.